game/squadrons/squadron.py

```diff
@@ -214,6 +214,10 @@ def _pilots_without_status(self, status: PilotStatus) -> list[Pilot]:
     def max_size(self) -> int:
         return self.settings.squadron_pilot_limit
 
+    @property
+    def expected_pilots_next_turn(self) -> int:
+        return len(self.active_pilots) + self.settings.squadron_replenishment_rate
```

Comment:
i think this has to use
```
min(
   self.settings.squadron_replenishment_rate,
   self._number_of_unfilled_pilot_slots,
)
```
better would be to make a property for it as well like
```
@property
def replenish_rate() -> int:
return min(...)
```

otherwise the `expected_pilots_next_turn` will return more pilots than there actually would be (replenish rate > pilots required)

Reply:
Good catch :)

Reply:
Thanks for noticing this! Has been modified, as suggested. :)